runtime/vm/LayoutFFITypeHelpers.hpp

```diff
@@ -179,14 +220,14 @@ class LayoutFFITypeHelpers
 	}
 
 	/**
-	 * @brief Create a FFI type from a layout string object for both primitive and struct
+	 * @brief Create a FFI type from a layout string object for both primitive and struct.
 	 *
 	 * @param typeFFI[in] The custom FFI type to be created
```

Comment:
I think `typeFFI` is an "out" parameter.

Reply:
Fixed.